modules/custom/dkan_harvest/src/Transform/AmpOust.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\dkan_harvest\Transform;
+
+use Harvest\ETL\Transform\Transform;
+
+/**
+ * Replaces "&" character with "and" in dataset title and themes.
+ */
+class AmpOust extends Transform {
+
+  /**
+   * Inherited.
+   *
+   * {@inheritDoc}
+   */
+  public function run($item) {
+
+    if ($item->theme) {
+      $counter = 0;
+      foreach ($item->theme as $key => $value) {
+        $item->theme[$key] = str_replace("&", "and", $item->theme[$key]);
+        $counter++;
```

Comment:
Seems like `$counter` is incremented but not used anywhere in this function. Consider removing.